src/osvr/Common/ImagingComponent.cpp

```diff
@@ -71,9 +70,15 @@ namespace common {
 
             template <typename T>
             void allocateBuffer(T &, size_t bytes, std::true_type const &) {
-                m_imgBuf.reset(reinterpret_cast<OSVR_ImageBufferElement *>(
-                                   cv::fastMalloc(bytes)),
-                               &cv::fastFree);
+                // Allocate a memory pointer aligned to a boundary.
+                size_t space = bytes + sizeof(void*) + OSVR_ALIGN_SIZE;
+                void* buffer = malloc(space);
+                void* aligned = std::align(OSVR_ALIGN_SIZE, bytes, buffer, space);
```

Comment:
I'm not terribly familiar with the C++11 alignment functionality yet - are we guaranteed here that `aligned > buffer` so that below, when we do `aligned[-1]` we're not writing out of bounds?